### ISSUE TITLE
The top line of a classic 2 table disappears when the table  is copied to Microsoft Word

### ISSUE THREAD

**Issue by jimingshuo** [haozhu233/kableExtra#540]
The top line of a classic 2 table disappears when the table  is copied to Microsoft Word document.

Also I would like to know what is the best way to insert an empty line between two tables. The [cat("\\  ")] command inserts a line like "## \ " in the output file.

**To Reproduce**
![Snipaste_2020-10-06_13-58-09](https://user-images.githubusercontent.com/33013981/95248084-64c89a80-07dc-11eb-8802-d0c903b9b039.png)
```
# code to reproduce the bug.
dt <- mtcars[1:5, 1:6]
dt %>%
  kbl(tab.cap = NULL, caption = "TABLE Test 1", align=c(rep('r', 10))) %>%
  kable_classic_2(bootstrap_options = c("condensed", "basic"), full_width = F, position = "left", font_size = 13)

cat("\\  ")
dt %>%
  kbl(tab.cap = NULL, caption = "TABLE Test 2", align=c(rep('r', 10))) %>%
  kable_classic_2(bootstrap_options = c("condensed", "basic"), full_width = F, position = "left", font_size = 13)
```

**Comment by haozhu233** [comment]
Yeah, I realized that word can only read css in cells. However, this has a little conflict with another issue which I couldn't find. Basically if you use collapse rows and some cells in the last row happens to be empty, there won't be a bottom border. Let me think about how to deal with that.
